application/views/admin/globalsettings/_email.php

```diff
@@ -64,9 +64,9 @@
                     'name' => 'emailsmtpssl',
                     'value'=> getGlobalSetting('emailsmtpssl') ,
```

Comment:
Maybe here, too?

Reply:
emailsmtpssl option was already lowercase, I didn't change the case for it, just the bounceencryption.